testCourse/questions/externalGrade/codeUpload/question.html

```diff
@@ -0,0 +1,26 @@
+<pl-question-panel>
+  <p>
```

Comment:
We may want to consider creating an image for the externalGrader that cannot be touched by anyone. If someone updates the image and changes something, the CI/CD tests will break and we will not know why. 

For optimization, it could be Alpine Linux, which will be much quicker in an already long pipeline. This should cut off 20-30 seconds at least in the test time.